addons/website_sale_wishlist/migrations/12.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,5 @@
+---Models in module 'website_sale_wishlist'---
+---Fields in module 'website_sale_wishlist'---
+website_sale_wishlist / product.wishlist         / session (char)                : DEL
```

Comment:
As this has been removed, you have to remove the entries that doesn't have `partner_id` or the `partner_id` is a public user (not sure if this is a real case - @Yajo can you say?).

Reply:
I see, it seems Odoo stores now `wishlist_ids` into the session object itself.

It's weird, I don't know how will they garbage-collect them now...

Well, anyway it seems like session-related wishlist products will be lost, so I guess the best way is to do what you say in a pre-migration, but also add `WHERE session IS NOT NULL`.